### PR TITLE
[SELS-TASK] Create user edit profile markup

### PR DESCRIPTION
**Pull request by jstephend-sun**
Asana [Task](https://app.asana.com/0/1201478050789792/1201478741328074/f)

expected output:
can navigate to /account/profile/edit
form validation (required, max 255 char)

update button
loading state
error message

commands:
```
cd frontend
npm i
npm start
```
```
cd backend
php artisan serve
```

prerequisites:
logged in user (route protection for auth users only)

screenshots:
image
image
image

### CODE REVIEW

File: frontend/src/components/UserProfile/UserProfileEdit.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  }, []);
+
+  const onSubmit = (data: FormInput) => {
+    setLoading(true);
+    setError(false);
+    backend.get('/sanctum/csrf-cookie').then((csrf_response) => {
+      backend
+        .put(`/api/users/${cookies.user.id}`, data)
+        .then((response: AxiosResponse) => {
+          console.log(response);
```
Comment (ejanton-sun) [inline frontend/src/components/UserProfile/UserProfileEdit.tsx]: remove console log

File: frontend/src/components/UserProfile/UserProfileEdit.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+          navigate('/');
+        })
+        .catch((err: AxiosError) => {
+          setError(true);
+          let errordata = err.response?.data;
+          console.log(errordata?.error?.message);
```
Comment (ejanton-sun) [inline frontend/src/components/UserProfile/UserProfileEdit.tsx]: remove console log

**Review by ejanton-sun — CHANGES_REQUESTED** [review]

File: frontend/src/components/UserProfile/UserProfileEdit.tsx (hunk 2 above)
Comment (jstephend-sun) [reply frontend/src/components/UserProfile/UserProfileEdit.tsx]: done

File: frontend/src/components/UserProfile/UserProfileEdit.tsx (hunk 1 above)
Comment (jstephend-sun) [reply frontend/src/components/UserProfile/UserProfileEdit.tsx]: done

**Review by ejanton-sun — APPROVED** [review]

**Pull request merged**